.github/workflows/build.yml

```
@@ -0,0 +1,23 @@
+name: Build through reusable workflow
+
+# This workflow will build the app.
+# It calls a reusable workflow developed by Ledger's internal developer team to build the application and upload the
+# resulting binaries.
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  build_application:
+    name: Build application using the reusable workflow
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_build.yml@xch/rust-ci
```

```suggestion
    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_build.yml@v1
```
To change once https://github.com/LedgerHQ/ledger-app-workflows/pull/38 is merged